go-gitea/gitea
# Support relative paths to videos from Wiki pages

This change fixes cases when a Wiki page refers to a video stored in the Wiki repository using relative path. It follows the similar case which has been already implemented for images.

Test plan:
- Create repository and Wiki page
- Clone the Wiki repository
- Add video to it, say `video.mp4`
- Modify the markdown file to refer to the video using `<video src="video.mp4">`
- Commit the Wiki page
- Observe that the video is properly displayed
